src/ring_theory/ideal/operations.lean

```diff
@@ -1604,3 +1604,105 @@ begin
 end
 
 end ring_hom
+
+namespace double_quot
+variables {R : Type u} [comm_ring R] (I J : ideal R)
+
+lemma add_comm : I+J = J+I := _root_.add_comm I J
+
+-- a few lemmas to help shorten the proofs later
+lemma left_proj_quot_add_mk (x :R) (hx : x ∈ I) : ideal.quotient.mk (I+J) x = 0 :=
+ideal.quotient.eq_zero_iff_mem.2 (ideal.mem_sup_left hx)
+
+lemma right_proj_quot_add_mk (x : R) (hx : x ∈ J) :  ideal.quotient.mk (I+J) x = 0 :=
+ideal.quotient.eq_zero_iff_mem.2 (ideal.mem_sup_right hx)
+
+lemma in_ker_proj_to_add_left : I.map(ideal.quotient.mk(I+J)) = ⊥ :=
+begin
+  simp_rw [ideal.map, ideal.span_eq_bot, set.mem_image],
+  rintros y ⟨x, hx, rfl⟩,
+  exact left_proj_quot_add_mk I J x hx,
+end
+
+
+lemma in_ker_proj_to_add_right : J.map(ideal.quotient.mk(I+J)) = ⊥ :=
+by {rw add_comm, apply in_ker_proj_to_add_left}
```

Comment:
I think these make more sense as a general lemma in the `ideal` namespace:
```lean
lemma map_mk_eq_bot_of_le (h : I ≤ J) : I.map (J^.quotient.mk) = ⊥ :=
begin
  simp_rw [ideal.map, ideal.span_eq_bot, set.mem_image],
  rintros y ⟨x, hx, rfl⟩,
  exact ideal.quotient.eq_zero_iff_mem.2 (h hx),
end
```
if you do that the proofs are trivial:
```
lemma in_ker_proj_to_add_left : I.map (I ⊔ J)^.quotient.mk = ⊥ :=
map_mk_eq_bot_of_le le_sup_left
```
which suggests you probably don't want them at all.

Reply:
Done !